seed/challenges/03-front-end-libraries/jquery.json

```diff
@@ -10,6 +10,31 @@
   ],
   "challenges": [
     {
+      "id": "5e7d674br67419c4cd512b9b",
+      "title": "Introduction to jQuery",
+      "description": [
+        [
+          "//i.imgur.com/hzACiml.png",
+          "The logo of jQuery that has three blue sound waves that increase in size with the text \"jQuery write less, do more.\" written on it.",
+          "jQuery is one of the many libraries for JavaScript. It is designed to simplify any scripting done on the client side.",
+          ""
+        ],
+        [
+          "//i.imgur.com/OHtxVL0.png",
+          "A screenshot of the general syntax. A dollar sign followed by the selector with a dot and an action.",
+          "jQuery's most characteristic feature is the dollar sign. Knowing how to use its potential you can easily manipulate elements, create animations and handle input events.",
```

Comment:
Change to:
"jQuery's most recognizable characteristic is the dollar sign syntax. With it, you can easily manipulate elements, create animations, and handle input events.